### ISSUE TITLE
Generate reimbursement

### ISSUE THREAD

**Issue by hansmariusoveras** [ESN-Trondheim/ESNbot#74]
Upload image of receipt and fill in some info and get a PDF

**Comment by sigmundhh** [comment]
I like the idea :)
What will be the output format for the PDF? Should it be the ESN reimbursement form?

**Comment by hansmariusoveras** [comment]
Yes, that is the idea, but perhaps the form can be updated to something that is easier to generate as long as the content remains the same?
